src/services/lnd/index.ts

```diff
@@ -651,7 +652,7 @@ export const LndService = (): ILightningService | LightningServiceError => {
       destination: decodedInvoice.destination,
       mtokens: milliSatsAmount.toString(),
       payment: decodedInvoice.paymentSecret as string,
-      max_fee: Number(maxFee),
+      max_fee: maxFee,
```

Comment:
just a question but not a blocker... is it more safe to include this even if we use routes? or is it completely ignored?

Reply:
So we have a maxFee passed into probeForRoute which captures this for payViaRoutes... but I was thinking to maybe even remove this in a follow-up PR (following bringing it up at ACE maybe)